\documentclass[11pt]{article} %indicating type of document, font size
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers
\usepackage{hyperref}
\usepackage[leftcaption]{sidecap} %package for adding side captions
\sidecaptionvpos{figure}{c} %position side caption
\usepackage{lineno} %package for adding line numbers
\usepackage{lipsum}
\usepackage{indentfirst}
\usepackage{color}
\usepackage{multirow}
\usepackage{booktabs}
\usepackage[pdftex]{graphicx}  
\usepackage[labelsep=period]{caption} %package for formatting figure captions, separation between figure number and caption is period per AJB formatting requirements
\usepackage{natbib} % package for formatting citations
\bibliographystyle{AJB} %indicating bibliography uses the AJB.bst file for formatting...journals often have these available but these can also be customized
\usepackage{authblk}
\usepackage[table]{xcolor} %for table line color alteration
\usepackage{subscript}
\renewcommand{\baselinestretch}{1.5}
\renewcommand{\refname}{\centering{REFERENCES}} %changing title of references to Literature Cited as stipulated by AJB

\newcommand{\jri}[1]{\textcolor{blue}{ \emph{\scriptsize  #1}} } %introducting new command "\jri" for jeff ross-ibarra to make comments in blue italics
\newcommand{\mbh}[1]{\textcolor{dandelion}{ \emph{\scriptsize  #1}} } %same for matt hufford in red
\newcommand{\lev}[1]{\textcolor{magenta}{\emph{\normalsize #1}} } %same for laura vann in green
\newcommand{\track}[1]{\textcolor{red}{\emph{\normalsize #1}} }


%----------------------------------------
%AUTHORS
%----------------------------------------

%\title{\LARGE{\bf{Natural variation in teosinte at the domestication locus \emph{teosinte branched1} (\emph{tb1})}}\footnotemark}


\author[1]{Authors: Laura Vann}%author information
\author[1,2]{Thomas Kono}
\author[1,3]{Tanja Pyh\"aj\"arvi}
\author[1,4,6]{Matthew B. Hufford}
\author[1,5,7]{Jeffrey Ross-Ibarra}
\affil[1]{Department of Plant Sciences, University of California Davis, Davis, CA, USA}
\affil[2]{Department of Agronomy and Plant Genetics, University of Minnesota Twin Cities, Minneapolis, MN, USA}
\affil[3]{Department of Biology, University of Oulu, Oulu, Finland}
\affil[4]{Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, Iowa, USA}
\affil[5]{Center for Population Biology and Genome Center, University of California Davis, Davis, CA, USA}
\affil[6]{Corresponding Author: Matthew B. Hufford; 339A Bessey Hall, Iowa State University, Ames, IA, USA; phone: 1-515-294-8511; email: mhufford@iastate.edu}
\affil[7]{Corresponding Author: Jeffrey Ross-Ibarra; 262 Robbins Hall, Mail Stop 4, University of California, Davis, CA, USA; phone: 1-530-752-1152; email: rossibarra@ucdavis.edu}
\date{}

\begin{document}

{\let\newpage\relax\maketitle}

\begin{linenumbers}
	\renewcommand\linenumberfont{\normalfont\bfseries\small}

\clearpage%inserts page break
%\footnotetext{\flushleft{Manuscript received \line(1,0){120};   revision accepted \line(1,0){120}.}
\vspace*{1cm}

%Code for SAS and command line stuff on Git
%Phenotyping on Figshare http://dx.doi.org/10.6084/m9.figshare.776926 -- do we need to cite this in the manuscript somewhere?


\begin{flushleft}


\parindent=20pt

\begin{centering}%centers section title
\section*{INTRODUCTION}
\end{centering}

Domesticated crops and their wild progenitors provide an excellent system in which to study adaptation and genomic changes associated with human-mediated selection \citep{Ross-Ibarra2007}. 
Plant domestication usually involves a suite of phenotypic changes such as loss of seed shattering and increased fruit or grain size, which are commonly referred to as the `domestication syndrome' \citep{OlsenWendel2013}, and much of the study of domestication has focused on understanding the genetic variation underlying these traits \citep{OlsenGross2010}. 
Because most domesticates show reduced genetic diversity relative to their wild counterparts, effort has been made to identify agronomically useful variation in crop wild relatives \citep{flint2009wide}. In some instances, the alleles conferring these beneficial traits are bred into domesticates for crop improvement. For example, \emph{Oryza rufipogon}, the wild progenitor of domesticated rice, has proven useful for the integration of a number of beneficial QTL controlling traits such as grain size and yield into domesticated rice \citep{KovachMcCouch2008}. In addition to researching the role of wild alleles in domesticates, researchers have also investigated the role of variation in domesticated taxa in the evolution of feral and weedy populations \citep{ellstrand2010crops}. But even though domesticated alleles are often found segregating in wild relatives \citep{Gallavotti2004, SigmonVollbrecht2010}, little is known about the ecological role of this variation in natural populations. In this paper we present an ecological genetic analysis of the domestication locus \emph{tb1}---specifically the domesticated haplotype at \emph{tb1}---in natural populations of the wild ancestor of domesticated maize. 

Maize (\emph{Zea mays} ssp. \emph{mays}) was domesticated from the teosinte \emph{Zea mays} ssp. \emph{parviglumis} (hereafter, \emph{parviglumis}) roughly 9,000 B.P. in southwest Mexico \citep{Piperno2009, Matsuoka2002}. Maize and the teosintes are an attractive system in which to study domestication due to the abundance of genetic tools developed for maize and well-characterized domestication loci \citep{Hufford2012a, Doebley2004, Hufford2012b}. 
\track{Additionally, large, naturally-occurring populations of both \emph{parviglumis} and \emph{Zea mays} ssp. \emph{mexicana} (hereafter, \emph{mexicana}) can be found throughout Mexico \citep{Wilkes1977, Hufford2013}. \emph{parviglumis}, with \emph{parviglumis} abundant in the lowlands of Mexico and \emph{mexicana} abundant in the highlands. Furthermore, both \emph{parviglumis} and \emph{mexicana} occur at high densities and genetic diversity of these taxa is estimated to be high \citep{Hufford2012a, Ross-Ibarra2009}}. 

Many morphological changes are associated with maize domestication, and understanding the genetic basis of these changes has been a focus of maize research for a number of years \citep{Doebley2004}. One of the most dramatic changes is found in plant architecture: domesticated maize is characterized by a central stalk with few tillers and lateral branches terminating in a female inflorescence, while teosinte is highly tillered and bears tassels (male inflorescences) at the end of its lateral branches. The \emph{teosinte branched1} (\emph{tb1}) gene, a repressor of organ growth, was identified as a major QTL involved in  branching \citep{DoebleyStecGustus1995} and tillering \citep{DoebleyStec1991} differences between maize and teosinte.  A 4.9 kb retrotransposon (\emph{Hopscotch}) insertion into the upstream control region of \emph{tb1} in maize acts to enhance expression of \emph{tb1}, thus repressing lateral organ growth \citep{Doebley1997, Studer2011}.  Dating of the \emph{Hopscotch} retrotransposon suggests that its insertion predates the domestication of maize, leading to the hypothesis that it was segregating as standing variation in populations of teosinte and increased to high frequency in maize due to selection during domestication \citep{Studer2011}. The effects of the \emph{Hopscotch} insertion have been studied in maize \citep{Studer2011}, and analysis of teosinte alleles at \emph{tb1} has identified functionally distinct allelic classes of \emph{tb1}\citep{StuderDoebley2012}, but little is known about the role of \emph{tb1} or the \emph{Hopscotch} insertion at this locus in natural populations of teosinte. Previous studies have confirmed the presence of the \emph{Hopscotch} in samples of \emph{parviglumis} and landrace maize \citep{Studer2011}; however, little is known about the frequency with which the \emph{Hopscotch} is segregating in natural populations.

In teosinte and other plants that grow at high population density, individuals detect competition from neighbors via the ratio of red to far-red light.  An increase in far-red relative to red light accompanies shading and triggers the shade avoidance syndrome, a suite of physiological and morphological changes such as reduced tillering, increased plant height and early flowering \citep{KebromBrutnell2007}. The \emph{tb1} locus appears to play an important role in the shade avoidance pathway in \emph{Zea mays} \citep{LukensDoebley1999} and other grasses \citep{KebromBrutnell2007} via changes in expression levels in response to shading. \citet{LukensDoebley1999} introgressed the teosinte \emph{tb1} allele into a maize inbred background and noted that under low density conditions plants were highly tillered but that under high density, plants showed significantly reduced tillers and grew taller. 
Based on these results we hypothesize that the \emph{Hopscotch} (\emph{i.e.}, the domesticated allele) at \emph{tb1} may play a role in the ecology of teosinte, especially in high-density populations. In this study we aim to characterize the distribution of the \emph{Hopscotch} insertion in \emph{parviglumis}, \emph{mexicana}, and landrace maize, and to examine the phenotypic effects of the insertion in \emph{parviglumis}. We use a combination of PCR genotyping for the \emph{Hopscotch} element in our full panel and sequencing of two small regions upstream of \emph{tb1} combined with a larger SNP dataset in a subset of teosinte populations to explore patterns of genetic variation at this locus. Finally, we test for an association between the \emph{Hopscotch} element and tillering phenotypes in samples from a natural population of \emph{parviglumis}.

\begin{centering}
\section*{MATERIALS \& METHODS}
\end{centering}

\textbf{Sampling and genotyping---}We sampled 1,110 individuals from 350 \track{populations} (247 maize landraces, 17 \emph{mexicana} populations, and 86 \emph{parviglumis} populations) and assessed the presence or absence of the \emph{Hopscotch} insertion (Table S1 and Table S2, See Supplemental Materials with the online version of this article). Numbers of individuals sampled \track{per population} ranged from 1-43 for \emph{parviglumis}, 1-35 for \emph{mexicana}, and 1-18 for the maize landrace populations. DNA was extracted from leaf tissue using a modified CTAB approach \citep{DoyleDoyle1990, Maloof1984}. We designed primers using PRIMER3 \citep{RozenSkaletsky2000} implemented in Geneious \citep{Kearse2012} to amplify the entire \emph{Hopscotch} element, as well as an internal primer allowing us to simultaneously check for possible PCR bias between presence and absence of the \emph{Hopscotch} insertion due to its large size ($\sim$5kb). Two PCRs were performed for each individual, one with primers flanking the \emph{Hopscotch} (HopF/HopR) and one with a flanking primer and an internal primer (HopF/HopIntR). Primer sequences are HopF, {\small 5'-TCGTTGATGCTTTGATGGATGG-3'}; 
HopR, {\small 5'-AACAGTATGATTTCATGGGACCG-3'}; and HopIntR, {\small  5'-CCTCCACCTCTCATGAGATCC-3'} (Fig. S1 and \track{\ref{Fig1Cartoon}}, See Supplemental Materials with the online version of this article). Homozygotes show a single band for absence of the element ($\sim$300bp) and two bands for presence of the element ($\sim$5kb, amplification of the entire element, and $\sim$1.1kb, amplification of part of the element), whereas heterozygotes show all three bands (Table S2, See Supplemental Materials with the online version of this article). Since we developed a PCR protocol for each allele, if only one PCR resolved well, we scored one allele for that individual rather than infer the diploid genotype. We used Phusion High Fidelity Enzyme (Thermo Fisher Scientific Inc., Waltham, Massachusetts, USA) and the following conditions for amplifications: 98$^{\circ}$C for 3 min, 30 cycles of 98$^{\circ}$C for 15 s, 65$^{\circ}$C for 30 s, and 72$^{\circ}$C for 3 min 30 s, with a final extension of 72$^{\circ}$C for 10 min. PCR products were visualized on a 1\% agarose gel and scored for presence/absence of the \emph{Hopscotch} based on band size.

\textbf{Genotyping analysis---}To calculate differentiation between populations (F\textsubscript{ST}) and subspecies (F\textsubscript{CT}) we used HierFstat \citep{Goudet2005}. These analyses only included populations ($n = 32$) in which eight or more chromosomes were sampled. To test the hypothesis that the \emph{Hopscotch} insertion may be adaptive under certain environmental conditions, we looked for significant associations between \emph{Hopscotch} frequency and environmental variables using the software BayEnv \citep{Coop2010}. BayEnv creates a covariance matrix of relatedness between populations and then tests a null model that allele frequencies in populations are determined by the covariance matrix of relatedness alone against the alternative model that allele frequencies are determined by a combination of the covariance matrix and an environmental variable, producing a posterior probability (\emph{i.e.,} Bayes Factor; \citealt{Coop2010}). We used teosinte (ssp. \emph{parviglumis} and ssp. \emph{mexicana}) genotyping and covariance data from \citet{Pyhajarvi2013} for BayEnv, with the \emph{Hopscotch} insertion coded as an additional biallelic marker. \track{SNP data from \citet{Pyhajarvi2013} were obtained using the MaizeSNP50 BeadChip and Infinium HD Assay (Illumina, San Diego, CA, USA) and phased using fastPHASE \citep{Scheet2006}. Environmental data were previously obtained for \citet{Pyhajarvi2013} from \url{www.worldclim.org} and soil data were downloaded from the Harmonized World Soil Database \citep{FAOHWSD} at \url{www.harvestchoice.org}.}  Environmental data represent average values for the last several decades (climatic data) or are likely stable over time (soil data) and therefore represent conditions important for local adaptation of our samples.  Information from these data sets was summarized by principle component analysis following \citet{Pyhajarvi2013}.

\textbf{Sequencing---}In addition to genotyping, we chose a subset of \emph{parviglumis} individuals for sequencing. We chose twelve individuals from each of four populations from Jalisco state, Mexico (San Lorenzo, La Mesa, Ejutla A, and Ejutla B). For amplification and sequencing, we selected two regions approximately 600bp in size from within the 5' UTR of \emph{tb1} (Region 1) and from 1,235bp upstream of the start of the \emph{Hopscotch} (66,169bp upstream from the start of the \emph{tb1} ORF; Region 2). We designed the following primers using PRIMER3 \citep{RozenSkaletsky2000}: for the 5' UTR, {\small 5-'GGATAATGTGCACCAGGTGT-3'} and {\small 5'-GCGTGCTAGAGACACYTGTTGCT-3'}; for the 66kb upstream region, {\small 5'-TGTCCTCGCCGCAACTC-3'} and {\small 5'-TGTACGCCCGCCCCTCATCA-3'} (Table S1, See Supplemental Materials with the online version of this article). We used Taq polymerase (New England Biolabs Inc., Ipswich, Massachusetts, USA) and the following thermal cycler conditions to amplify fragments: 94$^{\circ}$C for 3 min, 30 cycles of 92$^{\circ}$C for 40 s, annealing for 1 min, 72$^{\circ}$C for 40 s, and a final 10 min extension at 72$^{\circ}$C. Annealing temperatures for Region 1 and Region 2 were 59.7$^{\circ}$C and 58.8$^{\circ}$C, respectively. To clean excess primer and dNTPs we added two units of Exonuclease1 and 2.5 units of Antarctic Phosphatase to 8.0 $\mu$L of amplification product. This mix was placed on a thermal cycler with the following program: 37$^{\circ}$C for 30 min, 80$^{\circ}$C for 15 min, and a final cool-down step to 4$^{\circ}$C. 

We cloned cleaned fragments into a TOPO-TA vector (Life Technologies, Grand Island, New York, USA) using OneShot TOP10 chemically competent \emph{E. coli} cells, with an extended ligation time of 30 min for a complex target fragment. We plated cells on LB agar plates containing kanamycin, and screened colonies using vector primers M13 Forward and M13 Reverse under the following conditions: 96$^{\circ}$C for 5 min; then 35 cycles at 96$^{\circ}$C for 30 s, 53$^{\circ}$C for 30 s, 72$^{\circ}$C for 2 min; and a final extension at 72$^{\circ}$C for 4 min. We visualized amplification products for incorporation of our insert on a 1\% agarose TAE gel.

Amplification products with successful incorporation of our insert were cleaned using Exonuclease 1 and Antarctic Phosphatase following the procedures detailed above, and sequenced with vector primers M13 Forward and M13 Reverse using Sanger sequencing at the College of Agriculture and Environmental Sciences (CAES) sequencing center at UC Davis. We aligned and trimmed primer sequences from resulting sequences using the software Geneious \citep{Kearse2012}. Following alignment, we verified singleton SNPs by sequencing an additional one to four colonies from each clone. If the singleton was not present in these additional sequences it was considered an amplification or cloning error, and we replaced the base with the base of the additional sequences. If the singleton appeared in at least one of the additional sequences we considered it a real variant and kept it for further analyses. 

\textbf{Sequence analysis---}For population genetic analyses of sequenced Region 1 and sequenced Region 2 we used the  Libsequence package \citep{Thornton2003} to calculate pairwise F\textsubscript{ST} between populations and to calculate standard diversity statistics (number of haplotypes, haplotype diversity, Watterson's estimator $\hat\theta_W$, pairwise nucleotide diversity $\hat\theta_\pi$, and Tajima's D). To produce a visual representation of differentiation between sequences and examine patterns in sequence clustering by \emph{Hopscotch} genotype we used Phylip (\url{http://evolution.genetics.washington.edu/phylip.html}) to create neighbor-joining trees with bootstrap-supported nodes (100 repetitions). For creation of trees we also included homologous sequence data from Maize HapMapV2 \citep{Chia2012} for teosinte inbred lines (TILs), some of which are known to be homozygous for the \emph{Hopscotch} insertion (TIL03, TIL17, TIL09), as well as 59 lines of domesticated maize.

\textbf{Introgression analysis---}In order to assess patterns of linkage disequilibrium (LD) around the \emph{Hopscotch} element in the context of chromosomal patterns of LD we used Tassel \citep{Bradbury2007} and calculated LD between SNPs across chromosome 1 using previously published data from twelve plants each of the Ejutla A (EjuA), Ejutla B (EjuB), San Lorenzo (SLO), and La Mesa (MSA) populations \citep{Pyhajarvi2013}. We chose these populations because we had both genotyping data for the \emph{Hopscotch} as well as chromosome-wide SNP data for chromosome 1. For each population we filtered the initial set of 5,897 SNPs on chromosome 1 to accept only SNPs with a minor allele frequency of at least 0.1, resulting in 1,671, 3,023, 3,122, and 2,167 SNPs for SLO, EjuB, EjuA, and MSA, respectively. We then used Tassel \citep{Bradbury2007} to calculate linkage disequilibrium ($r^{2}$) across chromosome 1 for each population. 

We examined evidence of introgression on chromosome 1 in these same four populations (EjuA, EjuB, MSA, SLO) using STRUCTURE \citep{Falush2003} and phased data from \citet{Pyhajarvi2013}, combined with the corresponding SNP data from a diverse panel of 282 maize lines \citep{Cook2012}. SNPs were anchored in a modified version of the IBM genetic map \citep{Gerke2013}. Since STRUCTURE does not account for LD due to physical linkage we created haplotype blocks using a custom Perl script from \citet[][code available at \url{http://dx.doi.org/10.6084/m9.figshare.1165577}]{Hufford2013}.  In maize, LD decays over an average distance of 5500bp \citep{Chia2012}; because LD decay is even more rapid in teosinte \citep{Chia2012} we used a conservative haplotype block size of 5kb.  We ran STRUCTURE at K=2 under the linkage model, with the assumption being that individuals fall into either a maize or teosinte cluster, performing three replicates with an MCMC burn-in of 10,000 steps and 50,000 steps post burn-in.

\textbf{Phenotyping of \emph{parviglumis}---}To investigate the phenotypic effects of the \emph{Hopscotch} insertion in teosinte we conducted a phenotyping trial in which we germinated 250 seeds of \emph{parviglumis} collected in Jalisco state, Mexico (population San Lorenzo; \citealt{Hufford2010}) where the \emph{Hopscotch} insertion is segregating at highest frequency (0.44) in our initial genotyping sample set. In order to maximize the likelihood of finding the \emph{Hopscotch} in our association population we selected seeds from sites within the population where genotyped individuals were homozygous or heterozygous for the insertion. We chose between 10-13 seeds from each of 23 sampling sites. We treated seeds with Captan fungicide (Southern Agricultural Insecticides Inc., Palmetto, Florida, USA) and germinated them in petri dishes with filter paper. Following germination, 206 successful germinations were planted into one-gallon pots with potting soil and randomly spaced one foot apart on greenhouse benches. Plants were watered three times a day with an automatic drip containing 10-20-10 fertilizer, which was supplemented with hand watering on extremely hot and dry days. 

Starting on day 15, we measured tillering index as the ratio of the sum of tiller lengths to the height of the plant \citep{Briggs2007}. Following initial measurements, we phenotyped plants for tillering index every 5 days through day 40, and then on day 50 and day 60. On day 65 we measured culm diameter between the third and fourth nodes of each plant. 
Following phenotyping we extracted DNA from all plants using a modified SDS extraction protocol. 
We genotyped individuals for the \emph{Hopscotch} insertion following the PCR protocols listed above. 

Tillering index data for each genotypic class did not meet the criteria for a repeated measures ANOVA, so we transformed the data with a Box-Cox transformation ($\lambda=0$) in the Car Package for R \citep{FoxWeisberg2011} to improve the normality and homogeneity of variance among genotype classes.  We analyzed relationships between genotype and tillering index and tiller number using a repeated measures ANOVA through a general linear model function implemented in SAS v.9.3 (SAS Institute Inc., Cary, NC, USA). Additionally, in order to compare any association between \emph{Hopscotch} genotype and tillering and associations at other presumably unrelated traits, we performed an ANOVA between culm diameter and genotype using the same general linear model in SAS. Culm diameter is not believed to be correlated with tillering index or variation at \emph{tb1} and is used as our independent trait for phenotyping analyses. SAS code used for analysis is available at \url{ http://dx.doi.org/10.6084/m9.figshare.1166630}. 

\begin{centering}
\section*{RESULTS}
\end{centering}

\textbf{Genotyping for the \emph{Hopscotch} insertion---}The genotype at the \emph{Hopscotch} insertion was confirmed with two PCRs for 837 individuals of the 1,100 screened \track{(Table S1 and Table S2, See Supplemental Materials with the online version of this article).}Among the 247 maize landrace accessions genotyped, all but eight were homozygous for the presence of the insertion Within our \emph{parviglumis} and \emph{mexicana} samples we found the \emph{Hopscotch} insertion segregating in 37 ($n=86$) and four ($n=17$) populations, respectively, and at highest frequency within populations in the states of Jalisco, Colima, and Michoac\'{a}n in central-western Mexico (Fig. \ref{Fig2Map}). Using our \emph{Hopscotch} genotyping, we calculated differentiation between populations (F\textsubscript{ST}) and subspecies (F\textsubscript{CT}) for populations in which we sampled sixteen or more chromosomes. We found that F\textsubscript{CT} = 0, and levels of F\textsubscript{ST} among populations within each subspecies (0.22) and among all populations (0.23) (Table \ref{Table1Fst}) are similar to genome-wide estimates from previous studies \citealt{Pyhajarvi2013}. Although we found large variation in \emph{Hopscotch} allele frequency among our populations, BayEnv analysis did not indicate a correlation between the \emph{Hopscotch} insertion and environmental variables (all Bayes Factors $<$ 1). 

\textbf{Sequencing upstream regions of the \emph{tb1} ORF---}To investigate patterns of sequence diversity and linkage disequilibrium (LD) in the \emph{tb1} region and any evidence of selection on this locus, we sequenced two small ($<$1kb) regions upstream of the \emph{tb1} ORF in four populations \track{from the Jalisco region}. After alignment and singleton checking we recovered 48 and 40 segregating sites for the 5' UTR region (Region 1) and the 66kb upstream region (Region 2), respectively. For Region 1, Ejutla A has the highest values of haplotype diversity and $\theta_\pi$, while Ejutla B and La Mesa have comparable values of these summary statistics, and San Lorenzo has much lower values. Additionally, Tajima's D is strongly negative in the two Ejutla populations and La Mesa, but is closer to zero in San Lorenzo (Table \ref{Table2Diversity}). For Region 2, haplotype diversity and $\theta_\pi$, are similar for Ejutla A and Ejutla B, while La Mesa and San Lorenzo have slightly lower values for these statistics (Table \ref{Table2Diversity}). Tajima's D is positive \track{in all populations except La Mesa, where a slightly negative value suggests a slight excess of low frequency variants} (Table \ref{Table2Diversity}). Pairwise values of F\textsubscript{ST} within population pairs Ejutla A/Ejutla B and San Lorenzo/La Mesa are close to zero for both sequenced regions as well as for the \emph{Hopscotch}, while they are high for other population pairs (Table \ref{Table1Fst}). 

\textbf{Evidence of introgression around the \emph{tb1} region---} \track{We investigated the possibility of introgression as an explanation for the frequency of the \emph{Hopscotch} allele in populations of teosinte using previously collected SNP data from \citep{Pyhajarvi2013}.} The highest frequency of the \emph{Hopscotch} insertion in teosinte was found in \emph{parviglumis} sympatric with cultivated maize. Our initial hypothesis was that the high frequency of the \emph{Hopscotch} element in these populations could be attributed to introgression from maize into teosinte. To investigate this possibility we examined overall patterns of linkage disequilibrium across chromosome 1 and specifically in the \emph{tb1} region. If the \emph{Hopscotch} is found in these populations due to recent introgression from maize we would expect to find large blocks of linked markers near this element. We find no evidence of elevated linkage disequilibrium between the \emph{Hopscotch} and SNPs surrounding the \emph{tb1} region in our resequenced populations (Fig.\ref{Fig3LD}), and $r^{2}$ in the \emph{tb1} region  does not differ significantly between populations with (average $r^{2}$ of 0.085) and without (average $r^{2}=0.082$) the \emph{Hopscotch} insertion. In fact, average $r^{2}$ is lower in the \emph{tb1} region ($r^{2}=0.056$) than across the rest of chromosome 1 ($r^{2}=0.083$; Table \ref{Table3R2}).

Neighbor joining trees of our sequence data and data from the teosinte inbred lines (TILs; data from Maize HapMapV2, \citealt{Chia2012}) do not reveal any clear clustering pattern with respect to population or \emph{Hopscotch} genotype (Fig. S3, See Supplemental Materials with the online version of this article); individuals within our sample that have the \emph{Hopscotch} insertion do not group with the teosinte inbred lines or domesticated maize that have the \emph{Hopscotch} insertion. The lack of clustering of \emph{Hopscotch} genotypes in our NJ tree as well as the lack of LD around \emph{tb1} do not support the hypothesis that the \emph{Hopscotch} insertion in these populations of \emph{parviglumis} is the result of recent introgression. However, to further explore this hypothesis we performed a STRUCTURE analysis using Illumina MaizeSNP50 data from four of our \emph{parviglumis} populations (EjuA, EjuB, MSA, and SLO) \citep{Pyhajarvi2013} and the maize 282 diversity panel \citep{Cook2012}. 
The linkage model implemented in STRUCTURE can be used to identify ancestry of blocks of linked variants which would arise as the result of recent admixture between populations. If the \emph{Hopscotch} insertion is present in populations of \emph{parviglumis} as a result of recent admixture with domesticated maize, we would expect the insertion and linked variants in surrounding sites to be assigned to the ``maize" cluster in our STRUCTURE runs, not the ``teosinte" cluster. In all runs, assignment to maize in the \emph{tb1} region across all four \emph{parviglumis} populations is low (average 0.017) and much below the chromosome-wide average (0.20; Table \ref{Table4Q}; Fig. \ref{Fig4Structure}). 

\textbf{Phenotyping of \emph{Zea mays} ssp. \emph{parviglumis}---}To assess the contribution of \emph{tb1} to phenotypic variation in tillering in a natural population, we grew plants from seed sampled from the San Lorenzo population of \emph{parviglumis}, which had a high mean frequency (0.44) of the \emph{Hopscotch} insertion based on our initial genotyping. 
We measured tiller number and tillering index, the ratio of the sum of tiller lengths to plant height, for 206 plants from within the San Lorenzo population, and genotyped plants for the \emph{Hopscotch} insertion. 
We also measured culm diameter, a phenotype that differs between maize and teosinte but has not been shown to be affected by the \emph{Hopscotch} insertion \citep{Briggs2007}. Culm diameter is meant to be an independent trait against which we can compare patterns of tillering index x \emph{Hopscotch} genotype data. If tillering index in \emph{parviglumis} is affected by the \emph{Hopscotch} insertion, the expectation is that patterns of tillering index data will have a significant correlation with \emph{Hopscotch} genotype, whereas we should find no significant correlation between culm diameter and \emph{Hopscotch} genotype. 
Phenotypic data are available at \url{http://dx.doi.org/10.6084/m9.figshare.776926}. 
Our plantings produced 82 homozygotes for the \emph{Hopscotch} insertion at \emph{tb1}, 104 heterozygotes, and 20 homozygotes lacking the insertion; these numbers do not deviate from expectations of Hardy-Weinberg equilibrium. 
After performing a repeated measures ANOVA between our transformed tillering index data and \emph{Hopscotch} genotype we find no significant correlation between genotype at the \emph{Hopscotch} insertion and tillering index (Fig. \ref{Fig5Boxplots}), tiller number, or culm diameter. 
Only on day 40 did we observe a weak but statistically insignificant ($r^2= 0.02$, $p=0.0848$) correlation between tillering index and the \emph{Hopscotch} genotype, although in the opposite direction of that expected, with homozygotes for the insertion showing a higher tillering index.

\begin{centering}
\section*{DISCUSSION}
\end{centering}

Adaptation occurs due to selection on standing variation or \emph{de novo} mutations. Adaptation from standing variation has been well-described in a number of systems; for example, selection for lactose tolerance in humans \citep{Plantinga2012, Tishkoff2007}, variation at the \emph{Eda} locus in three-spined stickleback \citep{Kitano2008, Colosimo2005}, and pupal diapause in the Apple Maggot fly \citep{Feder2003}. Although the adaptive role of standing variation has been described in many systems, its importance in domestication is not as well studied. 

In maize, alleles at domestication loci (\emph{RAMOSA1}, \citealt{SigmonVollbrecht2010}; \emph{barren stalk1}, \citealt{Gallavotti2004}; and \emph{grassy tillers1}, \citealt{Whipple2011}) are thought to have been selected from standing variation, suggesting that diversity already present in teosinte may have played an important role in maize domestication. The \emph{teosinte branched1} gene is one of the best characterized domestication loci, and, while previous studies have suggested that differences in plant architecture between maize and teosinte are a result of selection on standing variation at this locus \citep{Clark2006, Studer2011}, much remains to be discovered regarding natural variation at this locus and its ecological role in teosinte.

\citet{Studer2011} genotyped 90 accessions of teosinte (inbred and outbred), providing the first evidence that the \emph{Hopscotch} insertion is segregating in teosinte. 
Given that the \emph{Hopscotch} insertion has been estimated to predate the domestication of maize, it is not surprising that it can be found segregating in populations of teosinte. However, by widely sampling across teosinte populations our study provides greater insight into the distribution and prevalence of the \emph{Hopscotch} in teosinte. While our findings are consistent with \citet{Studer2011} in that we identify the \emph{Hopscotch} allele segregating in teosinte, we find it at higher frequency than previously suggested.
Moreover, many of our \emph{parviglumis} populations with a high frequency of the \emph{Hopscotch} allele fall in the Jalisco cluster identified by \citet{Fukunaga2005}, and further distinguish this region from the Balsas River Basin where maize was domesticated \citep{Matsuoka2002}. Potential explanations for the high frequency of the \emph{Hopscotch} element in \emph{parviglumis} from the Jalisco cluster include gene flow from maize, genetic drift, and natural selection. 

While gene flow from crops into their wild relatives is well-known, \citep{Ellstrand1999, Zhang2009, Thurber2010, Baack2008, Hubner2012, Wilkes1977, VanHeerwaarden2011, Barrett1983}, our results do not suggest introgression from maize at the \emph{tb1} locus, and are more consistent with \citet{Hufford2013} who found resistance to introgression from maize into \emph{mexicana} around domestication loci. 
Clustering in our NJ trees does not reflect the pattern expected if maize alleles at the \emph{tb1} locus had introgressed into populations of teosinte.  Moreover, there is no signature of elevated LD in the \emph{tb1} region relative to the rest of chromosome 1, and Bayesian assignment to a maize cluster in this region is both low and below the chromosome-wide average (Fig. \ref{Fig4Structure}, Table \ref{Table4Q}). Together, these data point to an explanation other than recent introgression for the high observed frequency of \emph{Hopscotch} in the Jalisco cluster of our \emph{parviglumis} populations.

Although recent introgression seems unlikely, we cannot rule out ancient introgression as an explanation for the presence of the \emph{Hopscotch} in these populations. If the \emph{Hopscotch} allele was introgressed in the distant past, recombination may have broken up LD, a process that would be consistent with our data.  We find this scenario less plausible, however, as there is no reason why gene flow should have been high in the past but absent in present-day sympatric populations.  In fact, early generation maize-teosinte hybrids are common in these populations today (MB Hufford, pers. observation), and genetic data support ongoing gene flow between domesticated maize and both \emph{mexicana} and \emph{parviglumis} in a number of sympatric populations \citep{Hufford2013, Ellstrand2007, VanHeerwaarden2011, Warburton2011}. 

Remaining explanations for differential frequencies of the \emph{Hopscotch} among teosinte populations include both genetic drift and natural selection.  Previous studies using both SSRs and genome-wide SNP data have found evidence for a population bottleneck in the San Lorenzo population \citep{Hufford2010, Pyhajarvi2013}, and the lower levels of sequence diversity in this population in the 5' UTR (Region 1) coupled with more positive values of Tajima's D are consistent with these earlier findings. Such population bottlenecks can exaggerate the effects of genetic drift through which the \emph{Hopscotch} allele may have risen to high frequency entirely by chance. A bottleneck in San Lorenzo, however, does not explain the high frequency of the \emph{Hopscotch} in multiple populations in the Jalisco cluster. Moreover, available information on diversity and population structure among Jaliscan populations \citep{Hufford2010, Pyhajarvi2013} is not suggestive of recent colonization or other demographic events that would predict a high frequency of the allele across populations. 
Finally, diversity values in the 5' UTR of \emph{tb1} are suggestive of natural selection acting upon the gene in populations of \emph{parviglumis}. Overall nucleotide diversity is 76\% less than seen in the sequences from the 66kb upstream region, and Tajima's D is considerably lower and consistently negative across populations (Table \ref{Table2Diversity}). In fact, values of Tajima's D in the 5' UTR are toward the extreme negative end of the distribution of this statistic previously calculated across loci sequenced in \emph{parviglumis} \citep{Wright27052005, Moeller01072007}. Though not definitive, these results are consistent with the action of selection on the upstream region of \emph{tb1}, perhaps suggesting an ecological role for the gene in Jaliscan populations of \emph{parviglumis}.  Finally, while these results are consistent with selection at the \emph{tb1} locus in teosinte, they do not confirm selection specifically on the \emph{Hopscotch} insertion at this locus.

Significant effects of the \emph{Hopscotch} insertion on lateral branch length, number of cupules, and tillering index in domesticated maize have been well documented \citep{Studer2011}. \citet{Weber2007} described significant phenotypic associations between markers in and around \emph{tb1} and lateral branch length and female ear length in a sample from 74 natural populations of \emph{parviglumis} \citep{Weber2007}; however, these data did not include markers from the \emph{Hopscotch} region 66kb upstream of \emph{tb1}. Our study is the first to explicitly examine the phenotypic effects of the \emph{Hopscotch} insertion across a wide collection of individuals sampled from natural populations of teosinte. We have found no significant effect of the \emph{Hopscotch} insertion on tillering index or tiller number, a result that is discordant with its clear phenotypic effects in maize. \track{It is possible that the planting density of our seedlings (plants spaced 12" apart) was too high and led to an overall increase in tillering as previously seen in \citet{LukensDoebley1999}, which would have affected our measurements of tillering index.}

An alternative interpretation of this result would be that the \emph{Hopscotch} controls tillering in maize \citep{Studer2011}, but tillering in teosinte is affected by variation at other loci. Consistent with this interpretation, \emph{tb1} is thought to be part of a complex pathway controlling branching, tillering and other phenotypic traits \citep{KebromBrutnell2007, Clark2006}. 

A recent study by \citet{StuderDoebley2012} examined variation across traits in an allelic series study of the \emph{tb1} locus. \citet{StuderDoebley2012} introgressed nine unique teosinte \emph{tb1} segments (one from \emph{Zea diploperennis}, and four each from \emph{mexicana} and \emph{parviglumis}) into an inbred maize (W22) background and investigated their phenotypic effects. 
Their findings suggest that different teosinte \emph{tb1} segments produce equivalent effects on tillering and that variation in tillering observed across these taxa is not due to a \emph{tb1} allelic series but potentially due to variation at other, unlinked loci. Clues to the identity of these loci may be found in QTL studies that have identified loci controlling branching architecture (\emph{e.g.,} \citealt{DoebleyStec1991, DoebleyStec1993}). Many of these loci (\emph{grassy tillers}, \emph{gt1}; \emph{tassel-replaces-upper-ears1}, \emph{tru1}; \emph{terminal ear1}, \emph{te1}) have been shown to interact with \emph{tb1} \citep{Whipple2011, Li2012},  and both \emph{tru1} and \emph{te1} affect the same phenotypic traits as \emph{tb1} \citep{DoebleyStecGustus1995}. \emph{tru1}, for example, has been shown to act either epistatically or downstream of \emph{tb1}, affecting both branching architecture (decreased apical dominance) and tassel phenotypes (shortened tassel and shank length and reduced tassel number; \citealt{Li2012}). Variation in these additional loci may have affected tillering in our collections and contributed to the lack of correlation we see between \emph{Hopscotch} genotype and tillering. 

\begin{centering}
\section*{CONCLUSIONS}
\end{centering}

In conclusion, our findings demonstrate that the \emph{Hopscotch} allele is widespread in populations of \emph{parviglumis} and \emph{mexicana} and occasionally at high allele frequencies. 
Analysis of linkage using SNPs from across chromosome 1 does not suggest that the \emph{Hopscotch} allele is present in these populations due to recent introgression, and it seems unlikely that the insertion would have drifted to high frequency in multiple populations. 
We do, however, find preliminary evidence of selection on the \emph{tb1} locus in \emph{parviglumis}.  Coupled with our observation of high frequency of the \emph{Hopscotch} insertion in a number of populations, this suggests that the locus---and potentially the domestication allele at this locus---may play an ecological role in teosinte. 

In contrast to domesticated maize, the \emph{Hopscotch} insertion does not appear to have a large effect on tillering in a diverse sample of \emph{parviglumis} from a natural population and the phenotypic consequences of variation at \emph{tb1} thus remain unclear. Future studies should examine expression levels of \emph{tb1} in teosinte with and without the \emph{Hopscotch} insertion and further characterize the effects of additional loci involved in branching architecture (e.g. \emph{gt1}, \emph{tru1}, and \emph{te1}). These data, in conjunction with more exhaustive phenotyping, should help to further clarify the ecological significance of the domesticated \emph{tb1} allele in natural populations of teosinte.

\clearpage

\subsection*{Acknowledgements}%subsection header
The authors thank Graham Coop for helpful discussion and Lauryn Brown, Joshua Hegarty, Pui Yan Ho, and Garry Pearson for assistance with the phenotyping portion of this study.

\clearpage
\bibliography{refs}%print bibliography

\clearpage
% Table 1
\begin{table}[htbp]
\rowcolors{2}{gray!25}{white}%alternating row colors in table
  \centering
  \caption{Pairwise F\textsubscript{ST} values from sequence and \emph{Hopscotch} genotyping data}%table caption
    \begin{tabular}{cccc}\\\toprule%number of columns and line across top of table
    \textbf{Comparison} & \textbf{Region 1} & \textbf{Region 2} & \textbf{\emph{Hopscotch}} \\\midrule
    EjuA \& EjuB & 0     & 0     & 0 \\
    EjuA \& MSA & 0.326 & 0.328 & 0.186 \\
    EjuA \& SLO & 0.416 & 0.258 & 0.280 \\
    EjuB \& MSA & 0.397 & 0.365 & 0.188 \\
    EjuB \& SLO & 0.512  & 0.290 & 0.280 \\
    MSA \& SLO & 0.007  & 0 & 0.016 \\\bottomrule
    \end{tabular}
  \label{Table1Fst}%indicates link between table and mention in text
\end{table}

% Table 2 Diversity Statistics
\begin{table}[htbp]
\rowcolors{2}{gray!25}{white}
  \centering
  \caption{Population genetic statistics from resequenced regions near the \emph{tb1} locus}
    \begin{tabular}{ccccc}\\\toprule
    \textbf{Population} & \# \textbf{Haplotypes} & \textbf{Hap. Diversity} & \textbf{$\hat\theta_\pi$}    & \textbf{Tajima's D} \\\midrule
    \multicolumn{5}{c}{\textit{Region 1(5' UTR)}} \\
    \multicolumn{1}{c}{EJUA} & 8     & 0.859 & 0.005 & -1.650 \\
    \multicolumn{1}{c}{EJUB} & 5     & 0.709 & 0.004 & -1.831 \\
    \multicolumn{1}{c}{MSA} & 6     & 0.682 & 0.004 & -1.755 \\
    \multicolumn{1}{c}{SLO} & 3     & 0.318 & 0.001 & -0.729 \\
    \multicolumn{5}{c}{\textit{Region 2 (66kb upstream)}} \\
    \multicolumn{1}{c}{EJUA} & 8     & 0.894 & 0.018 & 0.623 \\
    \multicolumn{1}{c}{EJUB} & 8     & 0.894 & 0.016 & 0.295 \\
    \multicolumn{1}{c}{MSA} & 3     & 0.682 & 0.011 & -0.222 \\
    \multicolumn{1}{c}{SLO} & 4     & 0.742 & 0.014 & 0.932 \\\bottomrule
    \end{tabular}
  \label{Table2Diversity}
\end{table}

\clearpage

% Table 3 $r^{2}$ values
\begin{table}[htbp]
\rowcolors{2}{gray!25}{white}
  \centering
  \caption{mean $r^{2}$ values between SNPs on chromosome 1, in the broad \emph{tb1} region, within the 5' UTR of \emph{tb1} (Region 1), and 66kb upstream of \emph{tb1} (Region 2).}
    \begin{tabular}{ccccc}\\\toprule
    \textbf{Population} & \textbf{Chr. 1} & \textbf{\emph{tb1} region} & \textbf{Region 1} & \textbf{Region 2} \\\midrule
    Ejutla A & 0.095 & 0.050 & 0.747 & 0.215 \\
    Ejutla B & 0.069 & 0.051 & 0.660 & 0.186 \\
    La Mesa & 0.070 & 0.053 & 0.914 & 0.766 \\
    San Lorenzo & 0.101 & 0.067 & 0.912 & 0.636 \\
    \bottomrule
    \end{tabular}
  \label{Table3R2}
\end{table}

\clearpage

 % Table 4 Assignment to maize and teosinte in tb1 region and chr 1 
\begin{table}[htbp]
\rowcolors{2}{gray!25}{white}
  \centering
  \caption{Assignments to maize and teosinte in the \emph{tb1} and chromosome 1 regions from STRUCTURE}
    \begin{tabular}{ccccc} \
          & \multicolumn{2}{c}{\textbf{\emph{tb1} region}} & \multicolumn{2}{c}{\textbf{Chr 1}} \\\toprule
    \bf{Population} & \bf{Maize} & \bf{Teosinte} & \bf{Maize} & \bf{Teosinte} \\\midrule
    Ejutla A & 0.022 & 0.978 & 0.203 & 0.797 \\
    Ejutla B & 0.019 & 0.981 & 0.187 & 0.813 \\
    La Mesa & 0.012 & 0.988 & 0.193 & 0.807 \\
    San Lorenzo & 0.016 & 0.984 & 0.205 & 0.795 \\\bottomrule
    \end{tabular}
  \label{Table4Q}
\end{table}

\clearpage

%Figure 1 Locus Cartoon
%-------------------------------------------------------------------
\begin{figure*}[!t]
  \begin{center}
   \includegraphics[width=180mm]{Fig1LocusCartoon.png}
   \flushleft{Figure 1. Representation of the upstream regulatory region of \emph{tb1}, showing the \emph{tb1} coding region (green) and the \emph{Hopscotch} insertion (red). Arrows show the location of primer sets; in black, primers used for amplification and sequencing (Region 1; within the 5' UTR, and Region 2; 66,169 bp upstream from the tb1 ORF); in blue, primers used to genotype the \emph{Hopscotch} insertion.}
  \end{center}
\label{Fig1Cartoon}
\end{figure*}
%-------------------------------------------------------------------

\clearpage

%Figure 2 Map 
%--------------------------------------------
\begin{figure*}[!t]
\begin{center}
   \includegraphics[width=0.8\textwidth]{Fig2MapBalsas.png}
   \end{center}
    \caption{Map showing the frequency of the \emph{Hopscotch} allele in populations of \emph{parviglumis} where we sampled more than 6 individuals. Size of circles reflects number of individuals sampled. The Balsas River is shown, as the Balsas River Basin is believed to be the center of domestication of maize.} 
\label{Fig2Map}
\end{figure*}
%--------------------------------------------
\clearpage

% Figure 3. LD plots
%-------------------------------------------------------------------
\begin{figure*}[!t]
  \begin{center}
   \includegraphics[width=180mm]{Fig3LDPlots.png}
   \end{center}
    \caption{Linkage disequilibrium for SNPs in Mb 261-268 on chromosome 1. The yellow rectangle indicates the location of the \emph{Hopscotch} insertion and the green rectangle represents the \emph{tb1} ORF. A) Ejutla A; B) Ejutla B; C) La Mesa; D) San Lorenzo. The upper triangle above the black diagonal is colored based on the $r^{2}$ value between SNPs while the bottom triangle is colored based on p-value for the corresponding  $r^{2}$ value.}
\label{Fig3LD}
\end{figure*}
%-------------------------------------------------------------------
\clearpage

% Figure 4 STRUCTURE Plot 4 Pops
%-------------------------------------------------------------------
\begin{figure*}[!t]
  \begin{center}
   \includegraphics[width=160mm]{Fig4Structure.png}
     \end{center}
    \caption{STRUCTURE assignment to maize across a section of chromosome 1. The dotted lines mark the beginning of the sequenced region 66kb upstream (Region 2) and the end of the \emph{tb1} ORF.}
\label{Fig4Structure}
\end{figure*}
%-------------------------------------------------------------------
\clearpage

% Figure 5 Phenotyping BoxPlots
%-------------------------------------------------------------------
\begin{figure*}[!t]
  \begin{center}
   \includegraphics[width=160mm]{Figure5BoxplotPheno1.png}
   \end{center}
    \caption{Box-plots showing tillering index in greenhouse grow-outs for phenotyping. White indicates individuals homozygous for the \emph{Hopscotch}, light grey represents heterozygotes, and dark grey represents homozygotes for the teosinte (No \emph{Hopscotch}) allele. Within boxes, dark black lines represent the median, and the edges of the boxes are the first and third quartiles. Outliers are displayed as dots, the maximum value excluding outliers is shown with the top whisker, while the minimum excluding outliers is shown with the bottom whisker.}
\label{Fig5Boxplots}
\end{figure*}
%-------------------------------------------------------------------


\end{flushleft}

\end{linenumbers}

\end{document}

