\documentclass[11pt]{article}
\usepackage[letterpaper, margin=1in]{geometry} %package that allows changes in margins and header/footers
\newcommand{\res}[1]{\noindent \textcolor{blue}{{#1}} \\}
\newcommand{\jri}[1]{\noindent \textcolor{red}{{#1}} \\}
\newcommand{\lev}[1]{\noindent \textcolor{green}{{#1}} \\}
\usepackage{lipsum}
\usepackage{color}
\usepackage{multirow}
\renewcommand{\baselinestretch}{1.5}


\begin{document}

\noindent Dear Dr. Andrew Schnabel and anonymous reviewers, \\

We would like to thank the Editor in Chief and the two anonymous reviewers for their helpful comments and suggestions. We have addressed the comments to the text, but while we feel that the suggested new experiments would provide interesting results, we worry that they would require considerable extra time (many months) and are outside of the scope of this current work.  We have attached a revised version of our manuscript, including the suggested changes, and have detailed out our responses in blue below. We hope that you are willing to consider our revised manuscript for resubmission.\\\

\noindent Sincerely,\\

Jeffrey Ross-Ibarra and Matthew Hufford

\section*{Response to Associate Editor:}

1) I thought that the neighbor-joining analysis was of little value, because it was not clear to me just how the result could be expected to provide information about Hopscotch genotype.  Why would you expect individuals with similar Hopscotch genotypes to cluster together?  Also, although you don’t provide any information on the distribution of variable sites among accessions and taxa, your data set of only 48 and 40 segregating sites is almost certainly not sufficiently variable to resolve relationships among the very large number of accessions you were analyzing.  As a result, you produce mostly a huge polytomy, with a small amount of structure that appears to reflect the genetic similarities and differences already evident in Table 1.  The trees themselves are difficult for a reader to interpret, because none of the accession names at the branch tips (other than TIL) are defined in the text.\\

\jri{please add two backslashes after every reviewer comment for spacing. happy to remove trees, but perhaps better we redo trees collapsing identical haplotypes. or  make a haplotype network with pie charts or something.}
\res{I vote we take out trees}
\lev{I'd rather take them out. I don't think that they add a huge amount that isn't elsewhere in paper}

2) Phenotyping experiments - These are curious experiments.  First, why did you not generate a population of individuals with known genotypes and then test for the effect of genotype on phenotype?  Why take a random sample and just hope that you will get the necessary number of individuals of each genotype?  How can you randomize the growing conditions with respect to genotype, if you don’t know genotypes ahead of time but instead determine them after all the growing measurements are taken?\\

\res{We agree that a planned grow-out with previously determined genotypes would be useful, but were unable to do so due to our collections and available seed since samples used for phenotyping were were collected as seed from a naturally occuring population in Jalisco, Mexico.}

3) Why are sample sizes not reported?  You provide total samples sizes (eg, 206 or 216 for Phenotyping 1, depending on whether you read the Methods or the Results), but not sample sizes for each genotypic class.\\

\res{This has been addressed. We report accurate sample sizes for both phenotypings that are consistent throughout the text.}

4) It was not clear to me what “to detect the observed effect” means in lines 12-13 of p. 9.  Your Phenotyping 1 results show no effect of genotype on tillering.  Thus, it is not clear what effect you are referring to.\\

\res{We have clarified this in the text: "Based on these initial data, we conducted a \emph{post hoc} power analysis using effect size data for \emph{tb1} associated QTL from (Briggs \emph{et al.} 2007), which indicated that a minimum of 71 individuals in each genotypic class would be needed to detect the suggested effect of the \emph{Hopscotch} on tillering index from Briggs \emph{et al.} 2007."}

5) What was the purpose of the Phenotyping 2  experiment?  You already showed no effect of genotype on phenotype using populations where you are most likely to see segregation for the insertion, so it was not clear to me why sampling more broadly, especially from populations with no evidence of the Hopscotch insertion, would be expected to improve your understanding of this relationship.\\

\res{You are correct that our Phenotyping 1 did not show an effect of the Hopscotch on tillering index; however, the distribution of genotype classes was so heavily skewed to individuals with the Hopscotch insertion (both heterozygotes and homozygotes) that we felt we may not have had enough individuals without the insertion to have been able to detect an effect.}

\res{Our sampling for phenotyping experiments was limited to a population from which we had a large number of seed collected, and we were unable to survey more than one population for these experiments. For our more broad genotyping of both landrace maize, ssp. \emph{parviglumis}, and ssp. \emph{mexicana} we sampled as many population as we could. For most of these populations the prevalence of the Hopscotch was previously unknown, and we were interested in having an idea of the overall distribution and frequency of the Hopscotch throughout these locations.}

6) The discussion of introgression, genetic drift, and selection in your Discussion section seems to lack coherence.  It sometimes focuses on explaining the unexpectedly high frequency of Hopscotch in present-day populations, whereas other times it seems to be addressing the presence or absence of Hopscotch in teosinte in general, and it also sometimes seems to be addressing selection on the tb1 locus. 

\res{should we restructure this? I thought it read clearly. Key points were 1) Hop more widely spread than previously thought; 2) evidence of selection on Hop in teo}
\jri{clean up, restructure, thank editor, explain changes}



\section*{Reviewer 1}

%Reviewer \#1: In their paper 'Natural variation in teosinte at the domestication locus teosinte branched1 (tb1)', Vann et al survey a large sample of maize and teosinte individuals for the presence of the Hopscotch transposable element upstream of tb1. The authors find that Hopscotch is more widespread, across a large sampled area in Mexico, than previously thought, particularly in parviglumis. The authors go on to sequence regions up and downstream of Hopscotch in subset of their initial sample. In analyzing their sequence data, they find no evidence for recent introgression of the maize Hopscotch locus into wild teosintes. They also find evidence for selection acting on tb1 in parviglumis. In greenhouse experiments, no difference in tiller index or tiller number was observed between teosinte lines with or without Hopscotch. 

%\res{yes sounds right}

1) They present an interesting result, their conclusions are supported by their data, and the methods they use are appropriate. However, I feel like they need to make some further effort to understand what is happening mechanistically. What is Hopscotch doing in maize vs these sampled teosintes? I feel like the expression analyses and phenotyping experiments they suggest should be part of a later study should be part of this study. At a minimum, qRT-PCR experiments should be performed assessing tb1 expression in their with/without Hopscotch lines. Expression analyses of gt1, te1, and tru1 would be great, since they invoke variation at these loci to explain their results.

Another potential avenue, that the authors also suggest, lies in more directed and exhaustive greenhouse experiments. Variation in red and far-red light has been shown to be important in regulating both tb1 (in Sorghum) and gt1 in maize. Growth chamber experiments (with or without supplemental far red light, with or without Hopscotch), coupled to expression analyses in seedlings, seem like they should be possible.  In the greenhouse experiments that were done, phenotyping was incomplete. Apart from tillering, Hopscotch and natural variation at the tb1 locus appears to affect ear architecture and internode length. Was there any evidence for variation in these other traits? In the longer term, it would be nice to see what happens when you introgress these particular parviglumis loci into maize.\\ 

\res{Expression analyses of \emph{tb1} and additional loci coupled with a more exhaustive phenotyping would be interesting and useful in understanding the role of \emph{tb1} in branching architecture in teosinte, but is beyond the scope of our study. The suggested experiments would in themselves be enough for an additional paper. While we are unable to include these experiments in this paper we have text to the conclusions of this paper discussing their utility and the need for further work to elucidate the role of \emph{tb1} in branching architecture in teosinte: "Future studies should examine expression levels of \emph{tb1} in teosinte with and without the \emph{Hopscotch} insertion and further characterize the effects of additional loci involved in branching architecture (e.g. \emph{gt1}, \emph{tru1}, and \emph{te1}) as well as include a more exhaustive phenotyping including all traits.  These data, in conjunction with more exhaustive phenotyping, should help reveal the ecological significance of the domesticated \emph{tb1} allele in natural populations of teosinte."}

2) One minor issue: in Fig. 1, it would be nice to see where the Balsas River Basin is in relation to the sampling shown. 

\res{is this necessary?}
\jri{add to figure}

3) Reviewer 2: Although this paper brings up an interesting question regarding the role of tb1 standing variation in teosinte, it is not able to answer it completely.  It presents results that have been published previously about the presence of hopscotch in teosinte, maize, and mexicana.\\

\res{Reviewer 2 is correct in stating that it was known that the Hopscotch was present in some ssp. \emph{parviglumis}, ssp. \emph{mexicana}, and some landrace maize; however, previous studies did not sample as many populations or individuals per population as the study we present here. Our study is not only concerned with whether or not the \emph{Hopscotch} is present or absent in these populations but is also interested in what its frequency is so that we can better understand the evolutionary history of this insertion in teosinte. We have added clarifying text to the introduction: "The effects of the \emph{Hopscotch} insertion have been studied in maize (Studer \emph{et al.} 2011), and analysis of teosinte alleles at \emph{tb1} has identified functionally distinct allelic classes (Studer and Doebley 2012), but little is known about the role of \emph{tb1} or the \emph{Hopscotch} insertion in natural populations of teosinte. Previous studies have confirmed the presence of the \emph{Hopscotch} in samples of ssp. \emph{parviglumis}, ssp. \emph{mexicana}, and landrace maize; however little is known about the frequency with which the \emph{Hopscotch} is segregating in natural populations."}


4) Nevertheless, the manuscript is worth publishing as it does present data that may help rule out some possible explanations as to why tb1 is so common in parviglumus.\\

\res{Thank you; we hope that our paper in conjunction with future research and experiments can help to further the understanding of the role of both \emph{tb1} and the \emph{Hopscotch} insertion in teosinte.}

5) This paper is fairly maize-centric. A more general message is included in the discussion, not as much in the introduction, and not at all in the abstract. This could be easily remedied.\\

\jri{tweak intro and abstract, add a bit to discussion maybe, explain text changes here.}
\lev{Jeff and Matt, this is already in the intro. I don't see what more we can say that would be appropriate. ``The \emph{tb1} locus appears to play an important role in the shade avoidance pathway in \emph{Zea mays} and other grasses and may therefore be crucial to the ecology of teosinte."}

6) Methods and statistics were appropriately used as far as my expertise allows me to judge.

\res{Thank you.}. 

Specific suggestions follow:

7) Introduction:
Page 4, Lines 7 - 9: You say "Additionally, many domesticates show reduced genetic diversity when compared to their wild progenitors, and an understanding of the distribution of diversity in the wild and its phenotypic eﬀects has become increasingly useful to crop improvement (Kovach and McCouch, 2008).".  Can you tell us why an understanding is useful? Can you give some examples?\\

\res{We have expanded on this concept in the introduction using \emph{Oryza sativa} as an example: ``For example, \emph{Oryza rufipogon}, the wild progenitor of domesticated rice, has proven useful for the integration of a number of beneficial QTL controlling traits such as grain size and flowering time into domesticated rice (Kovach and McCouch 2008).}

8) Page 5, line 12 - 14: You say "The tb1 locus appears to play an important role in the shade avoidance pathway in Zea mays and other grasses and may therefore be crucial to the ecology of teosinte (Kebrom and Brutnell, 2007; Lukens and Doebley, 1999)." Please clarify; do the references belong to the first half of the sentence, supporting tb1 playing a role in Zea mays, or the second half of the sentence, supporting tb1 possibly being crucial to the ecology of teosinte? Which of the two distinct things have actually been studied? If the first, could you tell us how tb1 has been shown to work in maize under shade conditions? If the second, was it actually studied, or merely hypothesized in teosinte?\\ 

\res{We have clarified issues regarding the references for this section of the paper.}

9) Page 5, Lines 15 - 16: you say you ``aim to characterize the distribution of the Hopscotch insertion in parviglumis, mexicana, and landrace maize, and to examine the phenotypic eﬀects of the insertion in parviglumis."  The mexicana seem a bit tacked on and not much discussed. Did you phenotype mexicana? Can you present the data here, if so? Does the analysis of this group of teosintes add anything to your conclusions?\\

\res{Reviewer 2 is correct in noting that ssp. \emph{mexicana} is not as prevalent in our paper as ssp. \emph{parviglumis}. This fact is due to limitations in access to accessions of ssp. \emph{mexicana} as field collections for ssp. \emph{mexicana} are not nearly as extensive as those of ssp. \emph{parviglumis}. Additionally ssp. \emph{mexicana} are more difficult to cultivate in a greenhouse setting. We felt that it would be important to include ssp. \emph{mexicana} in our study, especially since it is known that gene flow occurs between ssp. \emph{mexicana} and landrace maize. We have noted in our methods that we were only able to phenotype a population from which we had a large number of seed collections.}

10) Materials and Methods:
Page 5, lines 22 - 23: how many individuals per accession? A fixed number, or variable, and depending on what? This can be mentioned so the reader does not have to go to the appendix or supplements.\\ 

\res{We have added a range of values (1-38 individuals) per population in the methods: ``We sampled 1,110 individuals from 350 accessions (247 maize landraces, 17 \emph{mexicana} populations, and 86 \emph{parviglumis} populations; ranging from 1-38 individuals per population) and assessed the presence or absence of the \emph{Hopscotch} insertion (Appendix 1 and Appendix 2, See Supplemental Materials with the online version of this article)."}

11) Page 5, Line 28: mention how big the hopscotch element is.  Can it be amplified in one amplicon? Again, this would be useful to know without having to look in appendix or supplements.\\

\res{The size of the amplicon is listed in the Materials and Methods following the primer sequences: ``Homozygotes show a single band for absence of the element ($\sim$300bp) and two bands for presence of the element ($\sim$5kb and $\sim$1.1kb), whereas heterozygotes are three-banded (Appendix 2, See Supplemental Materials with the online version of this article)."}

12) Page 6 line 6: "When only one PCR resolved well, we scored one allele for the individual".  Sorry, what does this mean? That you have one band (one assumes the short one?). If you are talking presence/absence, how do you know the allele?\\ 

\res{We have clarified in the text. The 2 PCRs are for the two different alleles. If only one PCR resolved we only included one allele for that individual instead of trying to impute a diploid genotype. We have included the following sentence in the text: ``Since we had a PCR for each allele, if only one PCR resolved well, we scored one allele for the individual."}

13) Page 7 lines 14 - 15 "These analyses only included populations in which eight or more individuals were sampled."  How many populations did that include?\\
 
\res{We included 32 populations for these analyses, and have edited the text to reflect this number: ``These analyses only included populations in which eight or more individuals were sampled, totally 32 populations."}

14) Page 7 lines 17 - 26: how many year's worth of environmental data went into your analysis? Can you hypothesize that tb1 differences can be correlated with current climactic data if the changes and subsequent selections must have happened many generations ago?\\
 
\res{The environmental daya used from Pyh{\"a}j{\"a}rvi \emph{et al.} (2013) was one year's worth of environmental data. A previous study by Hufford \emph{et al.} (2012) has shown that environmental conditions in areas where teosinte grows have not changed significantly over the last 10,000 years. We have clarified the use of one year's worth of data in the text: ``Environmental data (One year's worth) were obtained from \url{www.worldclim.org}, the Harmonized World Soil Database (FAOHWSD) and \url{www.harvestchoice.org} and summarized by principle component analysis following Pyh{\"a}j{\"a}rvi \emph{et al.} (2013)."}

15) Page 7 lines 22 - 23: "We used genotyping and covariance data from Pyhajarvi et al. (2013) for BayEnv" what kind of dataset is this? Teosinte?\\ 

\res{We have clarified the dataset in the paper: ``We used teosinte (ssp. \emph{parviglumis} and ssp. \emph{mexicana} genotyping and covariance data from Pyh{\"a}j{\"a}rvi \emph{et al.} (2013) for BayEnv, with the \emph{Hopscotch} insertion coded as an additional SNP. SNP data from Pyh{\"a}j{\"a}rvi \emph{et al.} (2013) were obtained using the MaizeSNP50 BeadChip and Infinium® HD Assay (Illumina, San Diego, CA, USA)."}

16) Page 8 lines 20 - 21:" We created haplotype blocks using a custom Perl script that grouped SNPs separated by less than 5kb into haplotypes." Does LD data in your parviglumus support 5Kb?\\ 

\res{hmm. Matt why did you use 5kb when you did this for your study?}
\jri{read Matt's paper, explain to reviewer}

17) Page 9 lines 2 - 3: "Plants were watered three times a day by hand and with an automatic drip containing 10-20-10 fertilizer." This sounds odd; by hand or automatically? Or you turned on the drip by hand? In which case, it was not automatic… \\

\res{We have edited the text to clarify: ``Plants were watered three times a day with an automatic drip containing 10-20-10 fertilizer, which was supplemented with hand watering on extremely hot and dry days."}

18) Page 9 lines 7 - 8: "Culm diameter is not believed to be correlated with tillering index or variation at tb1. F" So, why did you do it? Because of the last sentence in the materials and methods? Perhaps do not mention here that is not correlated; wait until the end of the materials and methods where you seem to explain it.\\ 

\res{Thank you for the suggestion, we have edited it in the text and added it to the end of the Materials and Methods section: ``Additionally, in order to compare any association between \emph{Hopscotch} genotype and tillering and associations at other presumably unrelated traits, we performed an ANOVA between culm diameter and genotype using the same general linear model in SAS. Culm diameter is not believed to be correlated with tillering index or variation at \emph{tb1} and is used as our independent trait for phenotyping analyses."}

19) Page 9 lines 9 - 10: "We genotyped individuals for the Hopscotch insertion following the protocols listed above." The PCR protocols or the sequencing protocols?\\ 

\res{We have clarified this in the manuscript.}

20) Results:
Page 10 lines 5 - 6: "Within our parviglumis and mexicana samples we found the Hopscotch insertion segregating in 37 and four populations, respectively"; remind us how many parviglumis and Mexicana samples you had total? So we can see if this comes out to a very different ratio than in maize.\\ 

\res{okay can put in ratios or something}
\jri{clarify in manuscript}

21) Page 10 lines 7 - 9: "Using our Hopscotch genotyping, we calculated diﬀerentiation between populations (FST) and subspecies (FCT) for populations in which we sampled eight or more alleles." What do you mean, 8 or more alleles? I thought you only had presence or absence; how are alleles differentiated with your PCR?\\ 

\res{We believe we addressed this with your previous comment concerning uncertainty of the PCRs we were using. As edited into the Materials and Methods section, there is a PCR that produces a specific size for each allele. We have edited subsequent text accordingly.}

22) Page 13 lines 25 - 27: "and genetic data support ongoing gene ﬂow between domesticated maize and both mexicana and parviglumis in a number of sympatric populations (Huﬀord et al., 2013; Ellstrand et al., 2007; van Heerwaarden et al., 2011). Add Warburton, et al., 2011.  Gene flow among different teosinte taxa and into the domesticated maize gene pool.  Genet Resour Crop Evol 58:1243-1261.\\ 

\res{We have added this reference as suggested.}

23) Page 14 line 15 to page 15 line 14: too long a paragraph. Break into two, perhaps at page 14 line 32.\\ 

\res{We have split the paragraph into two.}
\end{document}